terraform/eks/adot-operator/java_auto_instrumentation.tpl

```diff
@@ -0,0 +1,15 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: my-instrumentation
+  namespace: ${AOC_NAMESPACE}
+spec:
+  exporter:
+    endpoint: http://aoc-collector:${GRPC_PORT}
+  env:
+   - name: OTEL_SERVICE_NAME
+     value: ${SERVICE_NAME}
+   - name: OTEL_RESOURCE_ATTRIBUTES
+     value: "service.namespace=${SERVICE_NAMESPACE},service.name=${SERVICE_NAME}"
```

Comment:
setting `OTEL_SERVICE_NAME` and `service.name` in `OTEL_RESOURCE_ATTRIBUTES` is redundant. pick one of them.

Reply:
thanks. Updated. I have removed `service.name `from `OTEL_RESOURCE_ATTRIBUTES` as setting this doesn't set the `OTEL_SERVICE_NAME`. And `OTEL_SERVICE_NAME` is required to be set to ensure the metrics are exported to desired namespace in Cloudwatch.